input/postgres/statements.go

```diff
@@ -234,6 +236,22 @@ func GetStatements(server *state.Server, logger *util.Logger, db *sql.DB, global
 		if showtext {
 			statementTexts[key] = receivedQuery.String
 		}
+		if queryID.Valid && showtext {
+			server.QueryIdentitiesMutex.Lock()
+			if server.QueryIdentities == nil {
+				server.QueryIdentities = make(state.QueryIdentityMap)
+			}
+			if identity, ok := server.QueryIdentities[queryID.Int64]; ok {
+				identity.LastSeen = now
+			} else {
+				server.QueryIdentities[queryID.Int64] = state.QueryIdentity{
+					QueryID:     queryID.Int64,
```

Comment:
Don't we already have the queryID in the key? (since storage is at a premium here, it seems better to keep the struct as small as possible)

Reply:
Good point, there's no need to store the query ID twice.